devsetup/Makefile

```diff
@@ -60,6 +60,9 @@ EDPM_COMPUTE_ADDITIONAL_HOST_ROUTES ?= '[]'
 TLS_ENABLED ?= false
 DNS_DOMAIN ?= localdomain
 
+BARBICAN_SERVICE_ENABLED ?= false
```

Comment:
here we have default to false

in standalone.sh we have default to true (line 51/52):

```
BARBICAN_ENABLED=${BARBICAN_ENABLED:-true}
MANILA_ENABLED=${MANILA_ENABLED:-true}
```


can we be consistent on this

Reply:
Right, that's an error, it should default to true, as to not change the previous behavior